packages/itwinui-react/src/core/SearchBox/SearchBox.tsx

```diff
@@ -32,7 +32,7 @@ const SearchBoxContext = React.createContext<
       /**
        * Context prop for disabling subcomponents
        */
-      isDisabled?: boolean;
+      disabled?: boolean;
```

Comment:
breaking change

Reply:
Undid.
Who allowed me to add 'isDisabled' when every other component has "disabled" ???